core/src/main/java/tech/pegasys/eth2signer/core/Runner.java

```diff
@@ -164,6 +166,11 @@ private void registerPublicKeysRoute(
         .failureHandler(errorHandler);
   }
 
+  private void registerStaticRoute(final Router router) {
+    router.route("/*").handler(StaticHandler.create());
```

Comment:
Do we want to be a bit more selective here, just everything under swagger-ui/*?

Reply:
done, `/openapi/*` will be loaded from `openapi` local folder